frontend/src/main/java/starter/project/frontend/CarController.java

```diff
@@ -39,6 +40,7 @@ public Car getCar(@PathVariable("carId") int carId) {
 
     @PostMapping(path = "/cars", consumes = {"application/json"})
     public Map<Integer, Car> createCarList(@RequestBody List<Car> carList) {
+        List<Car> currentCarList = new ArrayList<>();
```

Comment:
DOn't need to initialise a new list here. What does it even do?

Reply:
resolved

Reply:
You don't need to reply resolved ;-) 
Also it's not really resolved until someone else has seen it and resolved it.

Reply:
kool